consent-ws/src/main/java/org/broadinstitute/consent/http/service/ontologyIndexer/IndexOntologyService.java

```diff
@@ -47,12 +51,12 @@ public String getIndexName() {
      * @throws IOException The exception
      */
     public void indexOntologies(List<StreamRec> streamRecList) throws IOException {
-
-        utils.validateIndexExists(client, indexName);
-
-        try {
+        try(RestClient client = getRestClient()) {
             for (StreamRec streamRec : streamRecList) {
 
+                // Deprecate everything that might already exist for this ontology file
+                utils.bulkDeprecateTerms(client, indexName, streamRec.getOntologyType());
+
```

Comment:
should we not deprecate until after we know the new ontology file is not problematic?